<launch>
	<arg name="path_file" default="path"/>
	<rosparam command="delete" param="/path" />
	<rosparam command="load" file="$(find rva_basic_tools)/test/$(arg path_file).yaml" />
	<node pkg="rva_basic_tools" type="pathPublisher.py" name="path_publisher">
		<param name="frame" value="odom"/>
	</node>
	<node pkg="rva_basic_tools" type="scan_downsampler.py" name="down" output="screen"/>
	<!-- <node pkg="rva_basic_tools" type="coll_avoidance2.py" name="coll" /> -->
	<node pkg="rva_basic_tools" type="coll_avoidance_pot_2.py" name="coll" output="screen">
		<param name="max_linear_speed" value="0.2"/>
		<param name="min_linear_speed" value="0.1"/>
		<param name="max_angular_speed" value="0.6"/>
	</node>
	<node pkg="rva_basic_tools" type="controlGoal.py" name="control">
		<param name="max_linear_speed" value="0.7"/>
		<param name="min_linear_speed" value="0.3"/>
		<param name="max_angular_speed" value="0.6"/>
		<param name="tolerancia_angulo" value="0.3"/>
		<param name="tolerancia_punto" value="0.25"/>
		<param name="tolerancia_destino" value="0.1"/>
		<!-- <param name="xgoal" value="3"/> -->
		<!-- <param name="ygoal" value="2"/> -->
	</node>
</launch>
